app/services/build_action_logger.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class BuildActionLogger
+  def initialize(build_action)
+    @build_action = build_action
+  end
+
+  def info(message, context: nil)
+    create_log(message, status: Constants::BuildAction::INFO, context: context)
+  end
+
+  def error(message, context: nil)
+    create_log(message, status: Constants::BuildAction::ERROR, context: context)
+  end
+
+  private
+
+  attr_reader :build_action
+
+  def create_log(message, status:, context:)
+    message_with_context = context.present? ? "[#{context}] #{message}" : message
```

Comment:
maybe it's better to move `context` into separate column? This way we'll be able to filter out logs for specific applications